### DISCUSSION TITLE
Raspberry Pi OS updates / Genmon updates

### DISCUSSION THREAD

**Discussion by Jkh62** [jgyates/genmon#1226] [General]
How often ( is it a good idea ) to update the Raspberry Pi OS ?
Genmon updates are automatic notification but I have not kept up on the OS change logs. to see what has been changed.

I am assuming if there is a major change or Genmon is coded for specific updates it will be posted in the forum.

Awesome update BTW and Happy New year to everyone !

**Comment by jgyates** [comment]
If you always want the latest security updates, then weekly. If you are wanting to maximize the life of your OS and genmon, then I would suggest at most quarterly. If you are updating genmon regularly I would suggest updating your OS at least semi regularly.  In my view it you will get a bit more life out of your OS. Considering Linux is a collection of a lot of software maintained by many different people you can possibly find yourself in a incompatible situation if you go too long without updating your OS. Genmon has to change when needed to support the latest changes and deprecations within the OS an python environment so new installs work, so keeping relatively close to the latest allows your environment to better match the testing environment.

**Reply by jgyates** [reply]
https://github.com/jgyates/genmon/wiki/Appendix-M-Tips-and-Tricks-useful-for-Genmon#updating-your-pi-system-software

**Comment by Jkh62** [comment]
Security is not really a concern ( for me ) since mine is not on the net.
Scripting support or related changes are the features that do make a difference .

I am sure any major change or revision required will show up in the forums and talked about.

Thank you for everything you do BTW.
